common/djangoapps/enrollment/api.py

```diff
@@ -18,14 +18,16 @@
 DEFAULT_DATA_API = 'enrollment.data'
 
 
-def get_enrollments(user_id):
+# Edraak: update function to accept request param
+def get_enrollments(user_id, request=None):
```

Comment:
Please avoid passing `request` all around the place. See the notes below.

Reply:
Done